data_generator.py

```diff
@@ -0,0 +1,36 @@
+import random
+
+first_names = ["James", "Mary", "Liam", "Olivia", "Robert", "Patricia", "Noah", "Emma", "John", "Jennifer"]
+middle_names = ["Louise", "Rose", "Grace", "William", "Jane", "Elizabeth", "Thomas", "Anne", "Ann", "David"]
+last_names = ["Smith", "Johnson", "Williams", "Brown", "Jones", "Garcia", "Miller", "Davis", "Rodriguez", "Martinez"]
+
+names = []
+
+for first_name in first_names:
+    for last_name in last_names:
+        names.append(first_name + ' ' + last_name)
+
+for first_name in first_names:
+    for middle_name in middle_names:
+        for last_name in last_names:
+            names.append(first_name + ' ' + middle_name + ' ' + last_name)
+
+countries = ["Switzerland", "Canada", "Sweden", "Australia", "United States", "Japan", "Germany", "New Zealand"]
+
+vaccines = [["Pfizer", "Mainz, Germany"], ["Moderna", "Massachusetts, U.S."], ["Novavax", "Maryland, U.S."]]

```

Comment:
These seem more like manufacturers than vaccines to me. Is the name of this list entirely accurate?

Reply:
I meant to refer to vaccines by the names Pfizer, Moderna, and Novavax.
But yes, provided along with the locations, they can be thought of manufacturers as well.
Hence made the changes.